lib/web_ui/dev/edge_installation.dart

```diff
@@ -68,12 +71,73 @@ Future<BrowserInstallation> getEdgeInstallation(
     // Since Edge is included in Windows, always assume there will be one on the
     // system.
     infoLog.writeln('Using the system version that is already installed.');
+    final EdgeLauncher edgeLauncher = EdgeLauncher();
+    if (edgeLauncher.isInstalled) {
+      infoLog.writeln('Launcher installation was skipped, already installed.');
+    } else {
+      infoLog.writeln('Installing MicrosoftEdgeLauncher');
+      await edgeLauncher.install();
+      infoLog.writeln(
+          'Installations complete. To launch it run ${edgeLauncher.executable}');
+    }
+
     return BrowserInstallation(
       version: 'system',
-      executable: PlatformBinding.instance.getCommandToRunEdge(),
+      executable: io.Directory(path.join(
+              edgeLauncher.launcherInstallationDir.path,
+              '${PlatformBinding.instance.getCommandToRunEdge()}'))
+          .path,
     );
   } else {
     infoLog.writeln('Unsupported version $requestedVersion.');
     throw UnimplementedError();
   }
 }
+
+class EdgeLauncher {
+  /// Url for downloading  `MicrosoftEdgeLauncher`.
+  ///
+  /// Only useful in Windows, hence not added to [PlatformBinding].
+  final String windowsEdgeLauncherDownloadUrl =
+      'https://github.com/MicrosoftEdge/edge-launcher/releases/download/1.2.0.0/MicrosoftEdgeLauncher.exe';
```

Comment:
Let's move the version number to `edge_lock.yaml`.

Reply:
done. btw this is not edge version, this is the launcher version.

Reply:
Cool. Yes, I imagine the launcher version will also change over time.

Reply:
I still see a hard-coded value